static/js/components/PostDisplay.js

```diff
@@ -26,9 +26,14 @@ const postTitle = (post: Post) =>
     >
       {post.title}
     </Link>
-    : <a className="post-title" href={post.url} target="_blank">
-      {post.title}
-    </a>
+    : <div>
+      <a className="post-title" href={post.url} target="_blank">
```

Comment:
Can you add `rel="noopener noreferrer"` to this?

Reply:
weird that the linter didn't catch that...

Reply:
Would be a good use case for a custom eslint plugin